service/controller/app/resource/releasemigration/create.go

```diff
@@ -43,6 +44,35 @@ func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
 		return nil
 	}
 
+	// Resource is used to migrating all charts. it's too noisy if it works on every reconciliation.
```

Comment:
This is the same case as the chartoperator resource. We only need it to execute for the chart-operator app CR.

Could you try and make the comment and logging similar? 

https://github.com/giantswarm/app-operator/blob/5d02ba5453a6dd119f0e39f8c8cf5698a286d501/service/controller/app/resource/chartoperator/create.go#L28-L34